build/NuSpecs/build-nupkg.ps1

```diff
@@ -108,11 +108,8 @@ function ConfigureNugetCommandLine {
 
 ConfigureNugetCommandLine("Microsoft.ProjectReunion")
 
-Copy-IntoNewDirectory -IfExists $BuildOutput\$BuildFlavor\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll "$runtimesDir\win10-x86\native"
 Copy-IntoNewDirectory -IfExists $BuildOutput\$BuildFlavor\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.pri "$runtimesDir\win10-x86\native"
```

Comment:
>Microsoft.ProjectReunion.pri [](http://example.com/codeflow?start=87&length=28)

what is this?

Reply:
Got me. It was there. PRI file for PRfwk? Do we need it? If we remove it will PRfwk contain a resources.pri?

Reply:
The only PRI in Reunion I know of comes from WinUI.

Reply:
How peculiar. But it's optional (-IfExists) so no harm if it's missing for now

If it's unnecessary we should drop it in the main branch. Will follow up